killer_queen/__init__.py

```diff
@@ -0,0 +1,15 @@
+from killer_queen.solver import Solver
+
+
+def main():
+    size = input("Enter the board size: ")
+    s = Solver(int(size))
+    solutions = s.solve()
+
+    for index, solution in enumerate(solutions):
+        print("\nSolution %s:" % (index + 1))
+        solution.display()
+
+
+if __name__ == '__main__':
```

Comment:
This line ensures this code is only run when executed from the command line. If this file is imported, `main()` is not run